src/lib/u2.js

```diff
@@ -1437,3 +1437,52 @@ foam.CLASS({
     }
   ]
 });
+
+foam.CLASS({
+  package: 'foam.u2',
+  name: 'CSS',
+
+  properties: [
+    {
+      class: 'String',
+      name: 'code'
+    },
+    {
+      name: 'name',
+      factory: function() { return 'CSS-' + this.$UID; }
+    }
+  ],
+
+  methods: [
+    function installInClass(cls) {
+      // There might be several CSS axioms on each class.
+      // Each one will try to run this code.
+      // In order to prevent them all from running, we check if cls.create
+      // has the __cssHandled property set.
+      if ( cls.create.__cssHandled ) return;
+
+      var oldCreate = cls.create;
+      var axiom = this;
+
+      cls.create = function(args, X) {
+        // Install our own CSS, and then all parent models as well.
+        var c = this;
+        while ( c && c.private_ ) {
+          if ( X.installedCSS[c.id] ) break;
```

Comment:
If a class has more than one create-decorating-axiom (like Singleton or Multiton, although, you probably wouldn't use those for Views, but something like them), then this could break.

Why not have each Axiom decorate create to install itself?  This would be the most common case and it would be the fastest and simplest.  If you have more than one axiom, then it gets decorated more than once.